.snapcraft/resources/backupdb

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [[ ${EUID} != 0 ]]; then
+    echo "[-] This task must be run with 'sudo'."
+    exit
+fi
+
+if $(ps x | grep "node ${SNAP}/main.js" | grep -qv "grep"); then
+    echo "[-] Please shutdown Rocket.Chat first to get a clean backup"
+    echo "[-] Use 'sudo systemctl stop snap.rocketchat-server.rocketchat-server'"
+    echo "[-] Backup aborted!"
+    exit
```

Comment:
While I agree the backups are cleaner. Mongo is perfectly capable of doing backups while under load.

I'm ok with the note being here... But I don't think it should be forced

Reply:
Alright, then I will remove the exit. I was just following the [guide](https://rocket.chat/docs/installation/manual-installation/ubuntu/snaps/#1-stop-your-rocketchat-server), even better if it's not necessary.